p2p/protocol/identify/obsaddr.go

```diff
@@ -69,7 +80,7 @@ func (oas *ObservedAddrSet) Add(addr ma.Multiaddr, observer ma.Multiaddr) {
 	// for zero-value.
 	if oas.addrs == nil {
 		oas.addrs = make(map[string]*ObservedAddr)
-		oas.ttl = pstore.OwnObservedAddrTTL
+		oas.ttl = OwnObservedAddrTTL
```

Comment:
I switched back and opened. https://github.com/libp2p/go-libp2p-peerstore/pull/13